conf/layer.conf

```diff
@@ -50,7 +50,7 @@ GPU_MEM_1024 = "396"
 LICENSE_FLAGS_WHITELIST += "commercial_faad2"
 
 # Enable early start of Thunder
-DISTRO_FEATURES_append = " compositor thunder"
+DISTRO_FEATURES_append = " compositor thunder securityagent pxcore-libnode"
```

Comment:
Please remove pxcore-libnode from the distro feature append. This shouldn't be added by us.

Reply:
Same for security agent, either enable it as part of the defaults or leave it out and let other layers include it in the package config. But I do not like using DISTRO features for individual components.